plugins/uefi-capsule/fu-uefi-nvram-device.c

```diff
@@ -40,6 +73,10 @@ fu_uefi_nvram_device_get_results(FuDevice *device, GError **error)
 		return TRUE;
 	}
 
+	/* capture EFI binary debug output */
+	if (fu_device_has_private_flag(device, FU_UEFI_DEVICE_FLAG_ENABLE_DEBUGGING))
+		fu_uefi_nvram_device_capture_debugging(device);
```

Comment:
I would think we want this even when GRUB is used too not just NVRAM.

Reply:
Ack, good catch. Fixed.